plugins/com.google.cloud.tools.eclipse.appengine.compat/src/com/google/cloud/tools/eclipse/appengine/compat/GpeMigrator.java

```diff
@@ -66,7 +66,8 @@
 
   // FacetedProject.METADATA_FILE = ".settings/" + FacetCorePlugin.PLUGIN_ID + ".xml";
   @VisibleForTesting
```

Comment:
Generally constants do not need to be @VisibleForTesting. You can always use the literal form.